main.js

```diff
@@ -0,0 +1,208 @@
+    e.preventDefault();
+
+    // Create popup container
+    const popupContainer = document.createElement('div');
+    popupContainer.classList.add('popup-container');
+
+    // Create popup content
+    const popupContent = document.createElement('div');
+    popupContent.classList.add('popup-content');
+
+    // Create close button
+    const closeButton = document.createElement('span');
+    closeButton.classList.add('close-button');
+    closeButton.textContent = 'X';
+
+    // Create H2 element
+    const h2Element = document.createElement('h2');
+    h2Element.textContent = programs[index].title;
+
+    // Create paragraph element
+    const paragraphElement = document.createElement('p');
+    paragraphElement.textContent = programs[index].description;
+
+    // Append H2, paragraph, and close button to popup content
+    popupContent.appendChild(closeButton);
+    popupContent.appendChild(h2Element);
+    popupContent.appendChild(paragraphElement);
+
+    // Append popup content to popup container
+    popupContainer.appendChild(popupContent);
+
+    // Append popup container to the body
+    document.body.appendChild(popupContainer);
+
+    // Add click event listener to close the popup
+    closeButton.addEventListener('click', () => {
+      // Remove the popup container
+      popupContainer.remove();
+    });
+  });
+});
+
+// person section
+
+const persons = [
+  {
+    name: 'Santiago',
+    image: 'images/santiago3.jpeg',
+    description: "Manages the company's financial operations, including budgeting, financial planning, and financial reporting.",
+    role: 'Chief Financial Officer (CFO)',
+  },
+  {
+    name: 'Catherine',
+    image: 'images/catherine4.jpeg',
+    description: 'Oversees the day-to-day operations of the company, ensuring efficient and effective processes across various departments.',
+    role: 'Chief Operations Officer (COO)',
+  },
+  {
+    name: 'Stellar',
+    image: 'images/stellar1.jpeg',
+    description: 'Responsible for aligning technology initiatives with business objectives, managing IT infrastructure, and driving digital transformation.',
+    role: 'Chief Technology Officer (CTO)',
+  },
+  {
+    name: 'Houda',
+    image: 'images/houda6.jpg',
+    description: 'Oversees all aspects of human resources, including recruitment, employee relations, training and development, and HR policies.',
+    role: 'Human Resources Director',
+  },
+];
+
+const section3 = document.querySelector('.section3');
+
+const div2 = document.createElement('div');
+div2.classList.add('div22');
+
+div2.innerHTML = `
+  ${persons
+    .map(
+      (person) => `
+      <div class="person">
+        <img class="personImage" src="${person.image}" alt="${person.name}_image">
+        <div class="clm2">
+          <h2 class="name">${person.name}</h2>
+          <p class="p1">${person.description}</p>
+          <div class="line2"></div>
+          <p class="p2">${person.role}</p>
+        </div>
+      </div>
```

Comment:
- [ Required ] Good job applying the changes requested by the previous reviewer in the speaker's section. However, I think you introduced a new issue while applying the changes. Please align the speaker card that appears to be out of flow with the other cards correctly. Please see the screenshot below for more details on this. I think this is caused by the extra markup in the speakers section in the `index.html` file.

## Your design
![Screenshot from 2023-07-07 13-13-06](https://github.com/Nessrine88/Capstone-Project/assets/16392046/96450db8-7291-4d92-b4d0-72a24ff66bcd)

## Behance design
![Screenshot from 2023-07-07 13-19-07](https://github.com/Nessrine88/Capstone-Project/assets/16392046/93f3deef-ee80-4a76-b878-7dbba46fb1b9)